### ISSUE TITLE
Some "as i32" casts missing in book code

### ISSUE THREAD

**Issue by daniel-tp** [thebracket/rustrogueliketutorial#83]
Some code, such as as the 3rd code block in chapter 7 (1.6 Taking Damage) hasn't been updated with the new usize changes, so copying and running the code doesn't work.

What it is:
```rust
pos.x = path.steps[1] % map.width;
pos.y = path.steps[1] / map.width; 
```

What it should be:
```rust
pos.x = path.steps[1] as i32 % map.width;  
pos.y = path.steps[1] as i32 / map.width; 
```

**Comment by thebracket** [comment]
Thanks! I'll try and catch as many of those as I can in the next push.

**Comment by MatteoGgl** [comment]
Just saw this issue, I fixed this problem in #102
